<?xml version="1.0"?>
<launch>
    <arg name="value" default="0"/>
    <group ns="aura">
    <node pkg="aura" type="aura_mapping" name="aura_mapping" output="screen" respawn="true">
        <param name="map_frame" type="string" value="/aura/map"/>
        <param name="odom_frame" type="string" value="/aura/odom"/>
        <param name="map_topic" type="string" value="base_map"/>
        <param name="pc_topic" type="string" value="/aura/point_cloud"/>
        <param name="hz" type="int" value="10"/>
        <param name="height" type="int" value="3"/>
        <param name="width" type="int" value="7"/>
        <param name="resolution" type="double" value="0.05"/>
        <param name="initial_x" type="double" value="-1"/>
        <param name="initial_y" type="double" value="-1.5"/>
        <param name="min_tolerance" type="double" value="0.05"/>
        <param name="max_tolerance" type="double" value="1"/>
        <param name="initial_value" type="int" value="$(arg value)" />
    </node>
    </group>
</launch>
